lib/sports_data_api.rb

```diff
@@ -3,6 +3,7 @@
 require "rest_client"
 require "time"
 require "multi_json"
+require "json"
```

Comment:
I'm pretty sure that `json` is already included anyway and this require just returns `false`

I think we can merge and remove that require after, I just made a branch on my fork with that change and all tests passed on travis